package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "lint": "eslint",
-    "spuild": "spuild frontend"
+    "spuild": "spuild frontend",
+    "build": "browserify -t vueify -e frontend/src/static/js/v_authorship.js -o frontend/src/build/v_authorship.js"
```

Comment:
add the build part to spuild as well?, so that dun need to run 2 commands?

Reply:
@ongspxm how do i add it in?

Reply:
Thanks @ongspxm !

Reply:
Seem to be getting an error with ```appveyor`` upon adding this tho @ongspxm